src/Server/data/servers.ts

```diff
@@ -1251,7 +1258,7 @@ export const serverMetadata: IServerMetadata[] = [
   {
     hackDifficulty: 30,
     hostname: "silver-helix",
-    literature: [LiteratureNames.NewTriads],
+    literature: [Literatures[LiteratureName.NewTriads].fn],
```

Comment:
This is kinda gross

Reply:
Yeah I agree. I'll try to think of a way that looks a bit better but especially with these Messages and Literatures it's difficult to get all the right type information to the Server object, since it's an enum that also has to go through typechecking functions (so directly assigning the value from the enum doesn't work).